lib/gs1/validations/date_validation.rb

```diff
@@ -12,7 +12,22 @@ def validate_date(_options = {})
       def valid_date?
         return true if data.is_a?(::Date)
 
-        ::Date.parse(data)
+        valid_format_a?(data) || valid_format_b?(data)
+      end
+
+      def valid_format_a?(data)
+        valid_date_format?(data, '%y%m%d')
+      end
+
+      # Read more about this date format in the GS1::ExpirationDate class.
+      def valid_format_b?(data)
```

Comment:
Är detta format generellt accepterat för datum i GS1 eller specifikt för `ExpirationDate`?

Reply:
`GS1::Extensions::MonthBasedDate` has now an own module, but I've let the validations share module.